packages/Search/src/details/DTK_DetailsPriorityQueue.hpp

```diff
@@ -51,19 +51,27 @@ class PriorityQueue
         // construct the element to compare to those in the queue
         T elem( std::forward<Args>( args )... );
 
-        // find position of the new element in the sorted array
-        // COMMENT: could consider implementing a binary search here
-        SizeType pos;
-        for ( pos = 0; pos < _size; ++pos )
-            if ( !_compare( _queue[pos], elem ) )
-                break;
+        // find position of the new element in the sorted array (binary search)
+        SizeType low = 0, high = _size;
```

Comment:
Should set `high = _size - 1`

Reply:
No, Imagine the queue is [2] and you want to insert 1. It would put it in front, I believe.
I also tried it, and the test crashed.